examples/assets/BUILD.bazel

```diff
@@ -1,6 +1,13 @@
 load("@aspect_rules_ts//ts:defs.bzl", "ts_config", "ts_project")
 load("@bazel_skylib//rules:build_test.bzl", "build_test")
 
+genrule(
+    name = "copy_example_json_file",
+    outs = ["src/example-copy.json"],
+    cmd = "cp ./$(location :example.json) $@",
```

Comment:
Maybe just do `echo 1 > $@` instead of copying a file? Just so we don't need the extra file checked in and the old vs new name etc.

```
genrule(
    name = "generated_json_file",
    outs = ["src/generated.json"],
    cmd = "echo 1 > $@",
)
```